android/app/src/main/java/app/organicmaps/MwmActivity.java

```diff
@@ -1267,26 +1264,20 @@ private boolean removeFragment(String className, boolean animate)
     return true;
   }
 
-  // Called from JNI.
   @Override
-  @SuppressWarnings("unused")
   public void onPlacePageActivated(@NonNull PlacePageData data)
   {
     // This will open the place page
     mPlacePageViewModel.setMapObject((MapObject) data);
   }
 
-  // Called from JNI.
   @Override
-  @SuppressWarnings("unused")
   public void onPlacePageDeactivated()
   {
     closePlacePage();
   }
 
-  // Called from JNI.
   @Override
-  @SuppressWarnings("unused")
```

Comment:
Is it intended? Isn't this code (and above) called from JNI?

In any case, merging it in a separate PR can be a safer approach (e.g. this large rename commit can be ignored in git history search).

Reply:
Yes, it's an interface implementation. Obviously this function is used

Reply:
Idk, my Android Studio had never complained about these lines. I guess when you allow AS scan the JNI code everything works fine. Sometimes JNI scan breaks and you have to retrigger it manually. Perhaps that was the issue